src/Middleware/Middleware.sln

```diff
@@ -135,7 +135,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Cors",
 EndProject
```

Comment:
We should remove the middleware.sln. It's already large enough that VS is having trouble loading it. It's also not useful as none of these projects reference eachother or are worked on together. Each middleware is it's own feature area and could have its own sln.

Reply:
Can we treat this as a separate issue?

Reply:
Sure